src/main/java/it/pagopa/ecommerce/commons/documents/v1/TransactionClosureData.java

```diff
@@ -0,0 +1,35 @@
+package it.pagopa.ecommerce.commons.documents.v1;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.lang.Nullable;
+
+/**
+ * Data related to Nodo close payment operation such as outcome (OK/KO)
+ */
+@AllArgsConstructor
+@Data
+@Document
+public class TransactionClosureData {
+
+    /**
+     * The Nodo closePayment outcome
+     */
+    @Nullable
```

Comment:
I add TransactionClosureData to both `CLOSED` and `CLOSURE_FAILED` for map here any additional data related to the Nodo closure process. By now only outcome is needed in order to perform refund or not when in `CLOSED` state and that state is not needed when in `CLOSURE_FAILED` because transactions in that state will not need to be refund since not authorized at all. This is the reason of nullability here but it can be put to not nullable and be saved for both `CLOSED` and `CLOSURE_FAILED` events

Reply:
For posterity, we keep the response outcome for `closePayment` as it is useful for two purposes: 
* if we receive a KO response from Nodo after a closePayment OK we won't receive a `sendPaymentResult` callback from Nodo (and thus can't notify the user), so we branch on this data (the only possible path forward would be to refund the transaction)
* we keep track of whether `UNAUTHORIZED` transaction (i.e. no authorization + closePayment KO) were followed by a OK or KO response from Nodo